cdap-app-fabric/src/main/java/io/cdap/cdap/support/services/SupportBundleService.java

```diff
@@ -64,12 +76,20 @@
 public class SupportBundleService implements Closeable {
```

Comment:
Seems like the SupportBundleService should be a well-defined interface with an implementation instead of a single class?

Reply:
no we don't need to achieve multiple inheritances